apps/notifications/src/panel/templates/summary-in-list.jsx

```diff
@@ -15,8 +15,7 @@ import ImagePreloader from './image-loader';
 
 import { html } from '../indices-to-html';
 
-var debug = require( 'debug' )( 'notifications:summary-in-list' );
-var { recordTracksEvent } = require( '../helpers/stats' );
+const { recordTracksEvent } = require( '../helpers/stats' );
```

Comment:
Could also be `import`. I guess some automated codemod missed it before, as it's not present in the initial `import` block.